docs/auth/android.md

```diff
@@ -19,19 +19,20 @@ Add the `RNFirebaseAuthPackage` to your `android/app/src/main/java/com/[app name
 
 ```java
 // ...
-import io.invertase.firebase.RNFirebasePackage;
+import com.facebook.react.ReactApplication;
 import io.invertase.firebase.auth.RNFirebaseAuthPackage; // <-- Add this line
```

Comment:
Rather than one for each package, maybe make the comment generic?
```suggestion
import io.invertase.firebase.auth.RNFirebaseAuthPackage; // <-- Add one line like this for each package you need
```

Reply:
although I guess if it's in every packages documentation it does need to be changed everywhere. RN60 was a big change...

Reply:
oh - that reminded me - you should mention specifically "this is for integration with react-native 0.60 and higher, for react-native <= 0.59 refer to previous versions of the documentation"

and the link should be like https://github.com/invertase/react-native-firebase-docs/blob/8c36d9df1a35e993f5d4126c33448256b9c5b682/docs/auth/android.md

Which I fished out by browsing history on the file and finding the last commit

Reply:
Yup good point - added that note with the link